sentry-core/build.gradle.kts

```diff
@@ -1,8 +1,15 @@
+import com.jfrog.bintray.gradle.tasks.BintrayPublishTask
+import com.jfrog.bintray.gradle.tasks.BintrayUploadTask
+import org.gradle.api.publish.PublishingExtension
+
 plugins {
     `java-library`
+    `maven-publish`
```

Comment:
@marandaneto I guess merging #70 will affect this bit

Reply:
not a problem